<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgUserCardTypeEdit</class>
 <widget class="QDialog" name="dlgUserCardTypeEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>122</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="flDateFrom">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lblDateFrom">
      <property name="text">
       <string>From: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="dtpDateFrom">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="flDateTo">
    <item row="0" column="0">
     <widget class="QLabel" name="lblDateTo">
      <property name="text">
       <string>To: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="dtpDateTo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="btbButtons">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledUnitTime">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledUnits">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>230</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblUnitTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time of unit: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUnits">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Units:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPrice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>27</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledPrice">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbValidDates">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>136</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Valid between dates ...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbValidDays">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>181</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Valid for specified number of days ...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledDays">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>200</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="lblValidDays">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>83</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of days: </string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ledName</tabstop>
  <tabstop>ledPrice</tabstop>
  <tabstop>ledUnits</tabstop>
  <tabstop>ledUnitTime</tabstop>
  <tabstop>rbValidDates</tabstop>
  <tabstop>dtpDateFrom</tabstop>
  <tabstop>dtpDateTo</tabstop>
  <tabstop>rbValidDays</tabstop>
  <tabstop>ledDays</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgUserCardTypeEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgUserCardTypeEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
